### DISCUSSION TITLE
(feature request) save all images when model is downloaded

### DISCUSSION THREAD

**Discussion by armandoalonso** [BlafKing/sd-civitai-browser-plus#152] [Ideas]
it would be super convenient if there was a setting you can enable that will save all images when the model is downloaded...

it will essentially automate clicking on save images when you click download, this will be very help when downloading a batch of models... right now in order to also get the images when multi-downloading models is to go into each model and click save images.

**Comment by BlafKing** [comment]
Thanks for the suggestion!

That sounds good to me!
I'll add a toggle in the settings in a future update that can be used to download all images automatically. :)

**Comment by BlafKing** [comment]
There is a now a setting in the CivitAI settings tab that can be used to toggle automatically downloading all images after a completed download in v3.3.0! :)
